lib/tools/adb-emu-commands.js

```diff
@@ -5,42 +5,46 @@ import path from 'path';
 
 let emuMethods = {};
 
-emuMethods.isEmulatorConnected = async function (udid) {
-  let emulators = await this.getConnectedEmulators();
-  if (!emulators.length) {
-    return false;
+emuMethods.getEmulators = async function() {
```

Comment:
It shouldnt disconnect but it might happen. I have never use `debounce` or `memoize`, no idea what difference between them and which one it's better. 
Should we cache the emulators list ? Or verify it's connected on every emu command ?